### DISCUSSION TITLE
Bookmarks/URLs back-up, syncing & sharing

### DISCUSSION THREAD

**Discussion by pabloski80** [immich-app/immich#3206] [Feature Request]
### The feature

Hello,
Would it be very difficult to extend the service with ability of storing offline, syncing and sharing the browser bookmarks URLs?

-I could imagine, they would be stored either in hierarchical tree- or computer folders-like structure, or flat category list. 
-Perhaps a single element could be stored in multiple categories, as in marked with multiple category_name tags.
-Perhaps an optional brief 160 character description could be attached to each URL.
-Perhaps optionally an on/off visibility of clusters of elements like Images/Videos/E-Documents (per folder or global setting) could be controlled by user.

All the best. Love the idea of the project.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is not in scope for Immich.

**Comment by pabloski80** [comment]
Ok. Thank You.
